x/executionlayer/client/rest/msgCreator.go

```diff
@@ -118,7 +119,7 @@ func bondUnbondMsgCreator(bondIsTrue bool, w http.ResponseWriter, cliCtx context
 		bondingUnbondingAbi = grpc.MakeArgsUnBonding(req.Amount)
 	}
 	paymentCode := grpc.LoadWasmFile(os.ExpandEnv("$HOME/.nodef/contracts/standard_payment.wasm"))
-	paymentAbi := grpc.MakeArgsStandardPayment(new(big.Int).SetUint64(req.GasPrice))
+	paymentAbi := grpc.MakeArgsStandardPayment(new(big.Int).SetUint64(req.Fee))
```

Comment:
Is this possible? Where is this req.Fee initialized?

Reply:
In bondReq struct:

```golang
type bondReq struct {
	ChainID              string `json:"chain_id"`
	TokenContractAddress string `json:"token_contract_address"`
	PubkeyOrName         string `json:"pubkey_or_name"`
	Amount               uint64 `json:"amount"`
	GasPrice             uint64 `json:"gas_price"`
	Fee                  uint64 `json:"fee"`
	Memo                 string `json:"memo"`
}
```

`Fee` can be collected from JSON request

Reply:
About the previous `GasPrice`, it was mistake. It should have been `Fee`